bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1263,6 +1275,11 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 >hgrid=ne0np4.ARCTICGRIS.ne30x8 maxpft=17 mask=tx0.1v2 use_cn=.false. use_crop=.false. irrigate=.true. glc_nec=10 do_transient_pfts=.false. use_excess_ice=.false.
 </init_interp_attributes>
 
+<init_interp_attributes sim_year="1979" use_cndv=".false." use_fates=".false." lnd_tuning_mode="clm6_0_cam7.0"
+                        hgrid="ne30np4.pg3" use_cn=".false." maxpft="17"
+>hgrid=ne30np4.pg3 maxpft=17 mask=t232 use_cn=.false. use_crop=.false. irrigate=.true. glc_nec=10 do_transient_pfts=.false. use_excess_ice=.false.
```

Comment:
Does not work, yet.

Reply:
@ekluzek I have not succeeded in getting this one to pick up the 1979 file. It picks up the 2000 file.

Everything else works.

Reply:
...so the ne30 for 1979 is the only one that still does not work.

Reply:
This is the new list of manual tests
```
SMS.f09_g17.I1850Clm60BgcCrop.derecho_intel.clm-clm60cam7LndTuningMode.20241009_165735_5zcae5
SMS.f09_g17.I2000Clm60BgcCrop.derecho_intel.clm-clm60cam7LndTuningMode.20241009_165703_jyf283

SMS.f19_g17.I1850Clm60Bgc.derecho_intel.clm-clm60cam7LndTuningMode.20241011_110211_pg70h3
SMS.f19_g17.I2000Clm60BgcCrop.derecho_intel.clm-clm60cam7LndTuningMode.20241009_164447_sip6kr

SMS.ne30pg3_t232.I1850Clm60BgcCrop.derecho_intel.clm-clm60cam7LndTuningMode.20241009_154244_qtkqsd
SMS.ne30pg3_t232.I2000Clm60BgcCrop.derecho_intel.clm-clm60cam7LndTuningMode.C.20241009_125311_vhh4br

SMS_Ln9.f09_g17.IHistClm60Sp.derecho_intel.clm-clm60cam7LndTuningMode_1979Start.20241010_155022_5d17pp
SMS_Ln9.f19_g17.IHistClm60Sp.derecho_intel.clm-clm60cam7LndTuningMode_1979Start.20241011_132006_nj2lt3
SMS_Ln9.ne30pg3_t232.IHistClm60Sp.derecho_intel.clm-clm60cam7LndTuningMode_1979Start.20241010_152949_1t42fb
```

Reply:
More about 1979 that I just looked into and that may be relevant. 
We generated only four 1979 fsurdat files for ctsm53, and they do not include f09, f19, ne30:
```
surfdata_ne0np4.ARCTICGRIS.ne30x8_hist_1979_78pfts_c240908.nc
surfdata_ne0np4.ARCTIC.ne30x4_hist_1979_78pfts_c240908.nc
surfdata_ne0np4CONUS.ne30x8_hist_1979_78pfts_c240908.nc
surfdata_ne0np4.POLARCAP.ne30x4_hist_1979_78pfts_c240908.nc
```

Reply:
Should I reverse any 1979 work that I did in this PR so far, and then make finidat changes just for the four grids that have 1979 fsurdat files?

Reply:
@slevis-lmwg CAM does need 1979 files for ne30, f09 and f19 as they do FHIST cases that start in 1979. Since they are FHIST they don't need to have surface datasets for 1979 as long as there are landuse.timeseries files which there are.

so is it just ne30 that fails for 1979 or f09 and f19 as well?

Reply:
(Sorry for the delayed response.) Only ne30 fails.